cvat/apps/dataset_manager/tests/assets/tasks.json

```diff
@@ -97,5 +97,18 @@
         ]
       }
     ]
+  },
+  "many jobs": {
```

Comment:
I don't understand for now why the task has many jobs?

Reply:
Also it has 5 frames. But the annotations file has frame 8.

Reply:
The issue describes a task with several segments. In this test, we get 13 frames divided into 3 segments.

Reply:
OK. Now I see.